core/app/views/refinery/admin/_menu.html.erb

```diff
@@ -1,4 +1,25 @@
-<nav id="menu">
-  <%= render partial: "refinery/admin/menu_item",
-             collection: current_refinery_user.active_plugins.in_menu -%>
-</nav>
+<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up"><i class="icon icon-bars"></i></button>
```

Comment:
I suggest if we're going to do this, lets do it well.  We should get a screen reader up and do a pass through the entire interface with that user's experience in mind.  Doing snippets here and there is just going to feel like patchwork to the visually impaired.